tests/test-distro.sh

```diff
@@ -2,9 +2,9 @@
 
 if echo "$*" | grep -q -- 'help'; then
     prog=$(basename -s.wrapper "$0")
-    echo "Usage: $prog LXC-IMAGE ORIGINAL-CHANNEL UPGRADE-WITH-CHANNEL [PROXY]"
+    echo "Usage: $prog LXC-IMAGE CONFINEMENT ORIGINAL-CHANNEL UPGRADE-WITH-CHANNEL [PROXY]"
```

Comment:
It is not a good idea to change positional arguments. This script is used by our CI so if we do this change we will get into trouble.